### DISCUSSION TITLE
ToC & Right Panel Customisation

### DISCUSSION THREAD

**Discussion by ghost** [cotes2020/jekyll-theme-chirpy#1853] [Ideas]
### Description

Honestly I was thinking if I should post in QnA but it looked more like an idea.

1st) Show ToC in a collapsed way at the beginning of post, only for smaller screens (specially phones) that doesn't have the right panel. And if the reader wants the ToC they can just click on it and the ToC unfolds.

2nd) When viewing posts in bigger screens have the ToC on top of recently updated and trending tags.

**Comment by kungfux** [comment]
1. This is tracked as #1664, which is now closed as implemented and will be available in a future release.
2. I don't think there are any plans to change this behavior since TOC stays on top while scrolling anyway.

**Reply by SoumyaK4** [reply]
Thanks for hearing my plea ☺️ 
Soumya Kanti Kundu 
[B&BStudio](https://weiqi.soumyak4.in)
